app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str
+                 ) -> None:
```

Comment:
```suggestion
    def __init__(
        self, 
        comfort_class: int, 
        clean_mark: int, 
        brand: str
     ) -> None:
```

Reply:
I believe this is the best way to deal with many argument in the function/method declaration